pkgs/by-name/e1/e1s/package.nix

```
@@ -0,0 +1,26 @@
+{ stdenv, lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "e1s";
+  version = "1.0.34";
+
+  src = fetchFromGitHub {
+    owner = "keidarcy";
+    repo = "e1s";
+    rev = "v${version}";
```

```suggestion
    rev = "refs/tags/v${version}";
```
This is to make sure we're getting the tag and not the branch.